compiler/damlc/daml-ide/src/DA/Daml/LanguageServer/Visualize.hs

```diff
@@ -0,0 +1,56 @@
+-- Copyright (c) 2019 The DAML Authors. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+-- | Execute Daml commands
```

Comment:
That description tells you less than the module name which at least mentions `Visualize`. Either make it more useful or just remove it.